src/group_theory/group_action/basic.lean

```diff
@@ -133,6 +133,12 @@ variables {α} {β}
       conv {to_rhs, rw [← hy, ← mul_one y, ← inv_mul_self x, ← mul_assoc,
         mul_action.mul_smul, hx]}⟩⟩)⟩
 
+@[to_additive] instance {b : β} : mul_action α (mul_action.orbit α b) :=
+{ smul := λ a b', ⟨a • b', mul_action.orbit_eq_iff.mp (eq.trans (mul_action.orbit_eq_iff.mpr
```

Comment:
Usually when defining a new action it's good practice to add a lemma that shows what it unfolds to. In this case, the lemma would be `orbit.coe_smul|